index.html

```diff
@@ -191,7 +121,7 @@ <h2 class="about-me-topper">About me</h2>
             </div>
 
             <div class="card framework-card">
-                <img src="images/frameworks.png" alt="framework_icon">
+                <img src="images/frameworks.png" alt="framework_icon" />
                 <div class="card-details">
                     <p>Frameworks</p>
```

Comment:
- [ ] Great job working on this milestone. In the mobile view, your framework list are justified but in the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=ZgI21TLlpHaIHveH-0) design they are aligned to the left. Kindly fix this issue.